script/build-external-libs/fetch-wownero.sh

```diff
@@ -10,10 +10,10 @@ cd $EXTERNAL_LIBS_BUILD_ROOT
 
 
 # url="$base_dir/vendor/wownero"
-url="https://github.com/wownero/wownero"
-# url="https://github.com/wowario/wownero"
-version="v0.5.0.2"
-# version="1b5cefa"
+# url="https://github.com/wownero/wownero"
+url="https://github.com/wowario/wownero"
+# version="v0.5.1.0"
+version="a7845ab"
```

Comment:
current commit is `1a54e7b`

Reply:
Yes, I'm testing different commits locally with a local repo. I think I found the problem, still building.